packages/api/tests/endpoints/test-endpoints-async-operations.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+const test = require('ava');
+
+const {
+  testUtils: { randomString }
+} = require('@cumulus/common');
+
+const asyncOperationsEndpoint = require('../../endpoints/async-operations');
+const {
+  AsyncOperation: AsyncOperationModel,
+  User
+} = require('../../models');
+const {
+  createFakeUser
+} = require('../../lib/testUtils');
+
+let asyncOperationModel;
+let userModel;
+let authHeaders;
+let context;
+
+test.before(async () => {
+  // Create AsyncOperations table
+  asyncOperationModel = new AsyncOperationModel({ tableName: randomString() });
+  await asyncOperationModel.createTable();
+
+  // Create Users table
+  userModel = new User(randomString());
+  await User.createTable(userModel.tableName, { name: 'userName', type: 'S' });
+
+  const authToken = (await createFakeUser({ userDbClient: userModel })).password;
+  authHeaders = {
+    Authorization: `Bearer ${authToken}`
+  };
+
+  context = {
```

Comment:
What is the purpose of using the context for testing? Does setting it as an env cause problems with parallel tests or something?

Reply:
Yeah, setting an environment variable affects all tests that are running at the same time.  `t.context` only applies to that particular test.